spacy_loggers/util.py

```diff
@@ -49,3 +52,34 @@ def dict_to_dot(obj: Dict[str, dict]) -> Dict[str, Any]:
     RETURNS (Dict[str, Any]): The key/value pairs.
     """
     return {".".join(key): value for key, value in walk_dict(obj)}
+
+
+def setup_custom_stats_matcher(
+    regexps: Optional[List[str]] = None,
+) -> Callable[[str], bool]:
+    try:
+        compiled = []
+        if compiled is not None:
```

Comment:
This `if` statement doesn't seem to be doing anything - did you mean to check `regexps` instead?

Reply:
Yup, good catch!